web/themes/custom/votegov/src/sass/uswds-overrides/usa-button.scss

```diff
@@ -3,6 +3,9 @@
 @use "mixins" as *;
 
 .usa-button {
+  margin-right: unset;
+  margin-inline-end: .5rem;
```

Comment:
We can remove this `margin-inline-end: .5rem;` I don't see it being used. Do you have an example of where it's needed?

Reply:
I included it so the component still matches how uswds created it but I don't see it being necessary either. I'll remove
<img width="597" alt="Screenshot 2024-07-08 at 3 35 21 PM" src="https://github.com/usagov/vote-gov-drupal/assets/121046395/519ea7b9-5188-4c1c-b076-408b4334a298">